core/src/main/java/com/microsoft/applicationinsights/internal/config/ConfigurationFileLocator.java

```diff
@@ -86,24 +85,26 @@ public InputStream getConfigurationFile() {
         }
 
         if (configurationFile != null) {
-            InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.INFO, "Configuration file has been successfully found in: '%s'", configurationFile);
+            InternalLogger.INSTANCE.info("Configuration file has been successfully found in: '%s'", configurationFile);
             try {
                 return new FileInputStream(configurationFile);
             } catch (FileNotFoundException e) {
-                InternalLogger.INSTANCE.logAlways(InternalLogger.LoggingLevel.WARN, "Configuration file '%s' could not be opened for reading", configurationFile);
-                InternalLogger.INSTANCE.trace("stack trace is %s", ExceptionUtils.getStackTrace(e));
+                InternalLogger.INSTANCE.warn("Configuration file '%s' could not be opened for reading %s", configurationFile,
```

Comment:
generally, i think it's a good idea to have a colon (`:`) before the stack trace printout.

Reply:
Sure will add a colon before stacktrace statements.